types/node/test/readline.ts

```diff
@@ -52,8 +53,8 @@ const rl: readline.ReadLine = readline.createInterface(new stream.Readable());
 }
 
 {
-    rl.question("query", (answer: string) => {});
```

Comment:
Please revert unrelated style changes, they are confusing to reviewers.

Reply:
I thought I configured my prettier to prevent this. My bad!

Reply:
The style change from double quotes to single was from prettier, and that is the intended setting for the repositories prettier, should I keep it or revert it anyway?

Reply:
I'm bumping this as this is something I can target when working on my next commit

Reply:
I would revert this to get the PR moving forward. It's a shame that we have an unenforced prettier config, so until we do actually enforce it and bulk fix everything, I would just go for the minimal diff.

Reply:
Oh, this is a really thread, my mistake. Seems like things have continued anyway.